collect_app/src/androidTest/java/org/odk/collect/android/support/pages/FirstLaunchPage.kt

```diff
@@ -1,25 +1,35 @@
 package org.odk.collect.android.support.pages
 
+import androidx.test.espresso.matcher.ViewMatchers.withSubstring
+import org.odk.collect.android.support.Interactions
+import org.odk.collect.strings.R.string
+
 class FirstLaunchPage : Page<FirstLaunchPage>() {
 
     override fun assertOnPage(): FirstLaunchPage {
-        assertText(org.odk.collect.strings.R.string.configure_with_qr_code)
+        assertText(string.configure_with_qr_code)
         return this
     }
 
     fun clickTryCollect(): MainMenuPage {
-        return tryAgainOnFail(MainMenuPage()) {
-            scrollToAndClickSubtext(org.odk.collect.strings.R.string.try_demo)
+        Interactions.clickOn(withSubstring(getTranslatedString(string.try_demo))) {
+            MainMenuPage().assertOnPage()
         }
+
+        return MainMenuPage()
     }
 
     fun clickManuallyEnterProjectDetails(): ManualProjectCreatorDialogPage {
-        scrollToAndClickText(org.odk.collect.strings.R.string.configure_manually)
-        return ManualProjectCreatorDialogPage().assertOnPage()
+        return clickOnString(
```

Comment:
This change should resolve [this flake](https://console.firebase.google.com/project/api-project-322300403941/testlab/histories/bh.f6f8331e5ae6e371/matrices/8553318839669939150/details?stepId=bs.9f00ed716c56931f&testCaseId=1&query=et.1dca92c466c6b500&deeplink=true).